src/Root_properties.m

```diff
@@ -3,43 +3,27 @@
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 %% REFERENCES
-function [Rl] = Root_properties(Rl, Ac, rroot, frac, bbx, KT)
-    %%% INPUTS
-    global DeltZ sfactor LAI_msr
-    % BR = 10:1:650; %% [gC /m^2 PFT]
-    % rroot =  0.5*1e-3 ; % 3.3*1e-4 ;%% [0.5-6 *10^-3] [m] root radius
-    %%% OUTPUTS
-    if KT < 2880
-        fr = 0.3 * 3 * exp(-0.15 * LAI_msr(KT)) / (exp(-0.15 * LAI_msr(KT)) + 2 * sfactor);
-        if fr < 0.15
-            fr = 0.15;
-        end
-    else
-        fr = 0.001;
-    end
-    % if KT<3840
-    % fr=-0.091*LAI(KT)+0.40;
-    % else
-    % fr=0.001;
-    % end
-    % elseif KT<3200
-    % fr=0.05;
-    % else
-    % fr=0.02;
-    % end
+%{
+    This function is used to calculate the dynamic growth of root
+    This part can refer to Wang et al. (2021) "Intergrated modelling of photosynthesis and transfer of energy, mass, and momentum in the SPAC system"
+%}
+
+function [Rl] = Root_properties(Rl, Ac, rroot, frac, bbx, KT, DeltZ, sfactor, LAI_msr)
+    %INPUTS
+    %global DeltZ sfactor LAI_msr

```

Comment:
```suggestion
function [Rl] = Root_properties(Rl, Ac, rroot, frac, bbx, KT, DeltZ, sfactor, LAI_msr)
%{
    This function is used to calculate the dynamic growth of root
    This part can refer to Wang et al. (2021) "Intergrated modelling of photosynthesis and transfer of energy, mass, and momentum in the SPAC system"
%}
```

Reply:
It would be nice if we can add an explanation for the input and output parameters.